go/client/cmd_simplefs_stat.go

```diff
@@ -97,21 +97,23 @@ func (c *CmdSimpleFSStat) Run() (err error) {
 
 		for _, r := range res.Revisions {
 			e := r.Entry
-			ui.Printf("%d)\t%s\t%s\t%d\t%s\t%s\n",
+			ui.Printf("%d)\t%s\t%s\t%d\t%s\t%s\t%s\n",
 				r.Revision, keybase1.FormatTime(e.Time),
 				keybase1.DirentTypeRevMap[e.DirentType],
-				e.Size, e.Name, e.LastWriterUnverified.Username)
+				e.Size, e.Name, e.LastWriterUnverified.Username,
+				keybase1.PrefetchStatusRevMap[e.PrefetchStatus])
```

Comment:
No need to have the explicit map lookup, since the `%s` will cause the existing `String()` method to be called.  (Here and below.)